src/smc_sagews/smc_sagews/sage_server.py

```diff
@@ -182,13 +182,32 @@ def recv(self):
             return 'blob', s[1:]
         raise ValueError("unknown message type '%s'"%s[0])
 
-TRUNCATE_MESG = "WARNING: Output truncated.  Type 'smc?' to learn how to raise the output limit."
 def truncate_text(s, max_size):
     if len(s) > max_size:
         return s[:max_size] + "[...]", True
     else:
         return s, False
 
+def truncate_text_warn(s, max_size, name):
+    r"""
+    INPUTS
```

Comment:
Your Sphinx formatting of the INPUTS, etc., here is pretty bad.  Try `sage_server.truncate_text_warn?` in a worksheet and get this: 
```
Docstring :
INPUTS - s - string to be truncated - max-size - truncation limit -
name - name of limiting parameter

OUTPUT string, possibly truncated did_truncate - boolean warning
message
```